test/runtests.jl

```diff
@@ -7,7 +7,7 @@ using Base.Test
 
 # Conversion and evaluation test
 @test m"expand((1+x)^2)" == m"1 + 2*x + x^2"
-@test mcall(:(exp(im*π))) == -1
```

Comment:
I've been wondering if we should just give up on unicode support as you've done here. Making it a barrier to passing the tests has kept me from shipping Maxima.jl with versions of maxima that don't support it... but maybe I should give up on that idea.